src/sage/schemes/cyclic_covers/cycliccover_finite_field.py

```diff
@@ -597,13 +597,13 @@ def _vertical_matrix_reduction(self, s0):
             [0 for i in range(d - 1)] + self._dflift.list() + [0 for i in range(d - 0)]
         )
 
-        rows = [f_co[d - 2 - i : -i - 1] for i in range(d - 1)]
-        rows += [fd_co[d - 1 - i : -i - 1] for i in range(d)]
+        rows = [f_co[d - 2 - i: -i - 1] for i in range(d - 1)]
+        rows += [fd_co[d - 1 - i: -i - 1] for i in range(d)]
```

Comment:
https://peps.python.org/pep-0008/#pet-peeves: "in a slice the colon acts like a binary operator, and should have equal amounts on either side (treating it as the operator with the lowest priority). "